node/src/main/scala/coop/rchain/node/node.scala

```diff
@@ -47,14 +48,18 @@ class NodeRuntime(conf: Configuration, host: String, scheduler: Scheduler) {
 
   private implicit val logSource: LogSource = LogSource(this.getClass)
 
+  val logger = Logger(this.getClass)
```

Comment:
`node.scala` returns a program that runs under the `Effect`, we should *NOT* introduce a logger here, we should reuse `Log[Effect]`. Yes, I know part of this code is imperative, but we should not contribute to its "imperativeness" even more. This was a fairly easy task (replace println with logs), pushing a bit of technical debt here seems logical